packages/sci-components/src/core/Accordion/__snapshots__/index.test.tsx.snap

```diff
@@ -1,16 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`<Accordion /> Test story renders snapshot 1`] = `
+exports[`<Accordion /> Default story renders snapshot 1`] = `
 <div
   class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation1 MuiAccordion-root MuiAccordion-gutters css-1x3fmw6-MuiPaper-root-MuiAccordion-root"
-  data-testid="accordion"
-  id="test-story"
 >
   <div
-    aria-controls="test-story-header-panel-content"
+    aria-controls="undefined-header-panel-content"
```

Comment:
not sure why the snapshot changed `test-story` to `undefined`. Seems like a bug 🤔 ? Thank you!

Reply:
Wow, great catch! Thank you for pointing that out. I'll look into it and get it fixed. 😯

Reply:
Found it 😬 it was due to a typo 😯